Modules/Brand/Http/Services/BrandService.php

```diff
@@ -79,7 +80,7 @@ public function createBrand(array $brandData): Brand
         $brandData["brand_key"] = 'bmc_' . Str::lower(Str::random(10));
 
         $slug = Str::slug($brandData["brand_name"], '-');
-        $count = Brand::where('brand_slug', $slug)->count();
+        $count = Brand::where(DB::raw('lower(brand_name)'), strtolower($brandData["brand_name"]))->count();
```

Comment:
this most probably will cause errors, better to do the same search with ```brand_slug``` field

Reply:
as we inserting slug like test,test-1,test-2 it will be appropriate to check the number of count of brand title